.mvn/extensions.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+  <extension>
+    <groupId>org.srcdeps.mvn</groupId>
+    <artifactId>srcdeps-maven-local-repository</artifactId>
+    <version>4.0.0</version>
+  </extension>
+  <extension>
+    <groupId>org.srcdeps.mvn</groupId>
+    <artifactId>srcdeps-maven-enforcer</artifactId>
+    <version>4.0.0</version>
+  </extension>
+</extensions>
```

Comment:
This is temporary to be able to build with SNAPSHOTs from source. Will be removed once all final versions are out.

Reply:
Is there a reason we can't release and use RC builds instead of snapshots? I really dislike the idea of snapshot dependencies in repositories. It makes the build unreliable and unreproducible between people.

Reply:
Basically I would say we could already release RC builds of all SmallRye projects and merge them into the `microprofile-4` branch, updating them to Final later..?

Reply:
Yes, this was before we had any MP spec release done and I was trying this out to check some early integration work. We can remove it now in favour of RCs.

Reply:
Actually, we still need to keep it because of RESTEasy due to a incompatible change in Config that breaks all RESTEasy, but I can remove the rest.